packages/Checkbox/src/Checkbox.styles.js

```diff
@@ -0,0 +1,125 @@
+import { css } from "styled-components";
+import stylers from "@paprika/stylers";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import tokens from "@paprika/tokens";
+
+const integize = token => Number.parseInt(token, 10);
```

Comment:
Should we add this to `stylers`? Maybe not in this PR, but when you have to figure out how to share styling for the `<Radio>`, I think it might be time to shuffle this out since it's pretty generic, and helps keep the code more concise.

Reply:
Not sure about the name `integize` though... 🤔

Reply:
😄 we can change when we refactor it for radio button

Reply:
@mikrotron didn't you add a `toInt` function somewhere?

Reply:
@alexzherdev no, just the `spacer()`, which eliminated _most_ of the cases where I needed to use `parseInt()`.  `toInt()` is probably a better name though.